src/modules/helpers/modifiers.ts

```diff
@@ -5,58 +5,32 @@ import { format } from '../../services/format.js';
 export function todo(message: string): void;
 export async function todo(
   message: string,
-  _cb?: () => Promise<unknown>
+  cb?: () => Promise<unknown>
 ): Promise<void>;
-export function todo(message: string, _cb?: () => unknown): void;
+export function todo(message: string, cb?: () => unknown): void;
 export async function todo(
   message: string | (() => unknown) | (() => Promise<unknown>),
   _cb?: (() => unknown) | (() => Promise<unknown>)
 ): Promise<void> {
   Write.log(
     `${indentation.hasDescribe ? '  ' : ''}${format(`● ${message}`).cyan().bold()}`
   );
-
-  /* c8 ignore start */ // Type guard
-  if (typeof _cb === 'function') {
-    const isAsync = _cb.constructor.name === 'AsyncFunction';
-
-    if (isAsync) return await Promise.resolve();
-    return;
-  }
-  /* c8 ignore stop */
 }
 
 export async function skip(
   message: string,
-  _cb: () => Promise<unknown>
+  cb: () => Promise<unknown>
 ): Promise<void>;
-export function skip(message: string, _cb: () => unknown): void;
-export async function skip(_cb: () => Promise<unknown>): Promise<void>;
-export function skip(_cb: () => unknown): void;
+export function skip(message: string, cb: () => unknown): void;
+export async function skip(cb: () => Promise<unknown>): Promise<void>;
+export function skip(cb: () => unknown): void;
 export async function skip(
   messageOrCb: string | (() => unknown) | (() => Promise<unknown>),
   _cb?: (() => unknown) | (() => Promise<unknown>)
 ): Promise<void> {
-  const message =
-    (typeof messageOrCb === 'string' && messageOrCb) || 'Skipping';
+  const message = typeof messageOrCb === 'string' ? messageOrCb : 'Skipping';
```

Comment:
This change is not related. Just a simplification.

Reply:
This logic is actually intended to prevent the user from passing an empty string. But I think that if it's the user's intention, I shouldn't interfere. LGTM.